src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -14,7 +14,7 @@ public static class SubsystemConstants {
 		public static final boolean APRILTAGS_ENABLED = false;
 		public static final boolean LIMELIGHT_ENABLED = false;
 		public static final boolean CLIMB_ENABLED = false;
-		public static final boolean LAUNCHER_ENABLED = false;
+		public static final boolean LAUNCHER_ENABLED = true;
```

Comment:
Should revert, typically we do not enable subsystems until they're physically ready on the robot.

Reply:
done, dw